Assets/Scripts/BaseScript.cs

```diff
@@ -49,9 +51,22 @@ public void RegisterFailedLanding()
         mainCamera.transform.localPosition = new Vector3(0, 5.717994f, -990);
 
         // Reset rocket
-        IRocket newRocket = rocket.CreateRocket(transform, new Vector3(0.02f, -1.89f, -92f));
+        //IRocket newRocket = rocket.CreateRocket(transform, new Vector3(0.02f, -1.89f, -92f));
```

Comment:
When I was thinking creating rocket from a prefab, I was thinking if there was a way to create the rocket directly from the prefabs we have in the assets file instead of passing in an existing game object. It looks like you're still passing in an initialized game object here, or is there something I'm misunderstanding?

Reply:
This line is commented out I thought.